spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -302,6 +304,142 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
+        # Get axial slices (S-I direction) with lesion for the midsagittal slice
+        axial_lesion_slices = np.unique(np.where(im_data[mid_sagittal_slice, :, :])[1])
+
+        if self.verbose == 2:
+            printv('  Mid-sagittal slice : ' + str(mid_sagittal_slice), self.verbose, type='info')
+
+        # --------------------------------------
+        # Compute tissue bridges for each axial slice
+        # --------------------------------------
+        tissue_bridges_dict = {}
+        # Iterate across axial slices to compute tissue bridges
+        for axial_slice in axial_lesion_slices:
+            # Get the column of the current slice
+            slice_data = im_data[mid_sagittal_slice, :, axial_slice]
+            # Get the spinal cord column of the current slice
+            slice_sc_data = im_sc_data[mid_sagittal_slice, :, axial_slice]
+            # Get the indices of the lesion in the column
+            lesion_indices = np.where(slice_data)[0]
+            # Get the indices of the spinal cord in the column
+            sc_indices = np.where(slice_sc_data)[0]
+
+            # Compute ventral and dorsal tissue bridges
+            dorsal_bridge_width = lesion_indices[0] - sc_indices[0]           # [0] returns the most dorsal elements
+            # if the lesion extends the spinal cord, the dorsal bridge is set to 0
+            if dorsal_bridge_width < 0:
+                dorsal_bridge_width = 0
+            ventral_bridge_width = sc_indices[-1] - lesion_indices[-1]        # [-1] returns the most ventral elements
+            # if the lesion extends the spinal cord, the ventral bridge is set to 0
+            if ventral_bridge_width < 0:
+                ventral_bridge_width = 0
+
+            tissue_bridges_dict[axial_slice] = {'dorsal_bridge_width': dorsal_bridge_width,
+                                                'ventral_bridge_width': ventral_bridge_width,
+                                                'lesion_indices': lesion_indices}
+
+        # --------------------------------------
+        # Get minimal tissue bridges
+        # --------------------------------------
+        # Convert the dictionary to a DataFrame (for easier manipulation)
+        tissue_bridges_df = pd.DataFrame.from_dict(tissue_bridges_dict, orient='index')
+
+        # Get minimum dorsal and ventral tissue bridges together with their corresponding slices (slices are indices)
+        # NOTE: we get minimum because tissue bridges are quantified as the width of spared tissue at the minimum
+        # distance from cerebrospinal fluid to the lesion boundary
+        min_dorsal_bridge_width_slice = tissue_bridges_df['dorsal_bridge_width'].idxmin()
+        min_ventral_bridge_width_slice = tissue_bridges_df['ventral_bridge_width'].idxmin()
+
+        # Get the width of the tissue bridges in mm
+        # TODO: verify whether to use [0] or [1] for the pixel size
```

Comment:
Can we know which index corresponds to the height and width of the pixel? Ideally, because we're traversing from P-A (A-P) to calculate the tissue bridge for each axial slice, I think we have to multiply with the _height_ of the pixel (to get the values in mm). 

Don't you think it should be [0] then in that case?

Reply:
Thanks, Naga! 
I debugged the code again, and I believe that `p_lst[1]` is okay. I clarified it in [this commit](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489/commits/93e7953d0e50313153adc47076d824058a93f6dc):
```
# Get the width of the tissue bridges in mm
# NOTE: the orientation is RPI (because we reoriented the image to RPI using orient2rpi()); therefore p_lst[0]
# is the pixel size in the R-L direction, p_lst[1] is the pixel size in the A-P direction, and p_lst[2] is the
# pixel size in the S-I direction.
# Since we are computing dorsal and ventral tissue bridges, we use p_lst[1] (A-P direction)
```

Reply:
wonderful! this clarification note was important! I agree, `p_lst[1]` should be the correct option!